app/models/ability.rb

```diff
@@ -34,4 +35,11 @@ def admin_abilities
 
     can :manage, :all
   end
+
+  def alumni_abilities
+    return unless @user.alumni?
+
+    can :read, User, id: @user.id
```

Comment:
It was me that told you to do this, but now that I'm looking at it - what if another role requires a `can :read, User, id: @user.id` - this would open it up to a user with that role as well. This needs to be specific for an Alumni. I did a bit more reading and think we should do
```can :alumni, User, id: @user.id```
`alumni` can be anything, but I wanted it to be specific to Alumni